src/bitmessageqt/foldertree.py

```diff
@@ -449,24 +510,34 @@ def setData(self, role, value):
                 self.label = str(value.toString().toUtf8())
             else:
                 self.label = str(value)
-            if self.type in (AccountMixin.NORMAL, AccountMixin.MAILINGLIST, AccountMixin.CHAN):
+            if self.type in (
+                    AccountMixin.NORMAL,
+                    AccountMixin.MAILINGLIST,
+                    AccountMixin.CHAN):
                 try:
-                    a = BMConfigParser().get(self.address, 'label')
+                    # a = BMConfigParser().get(self.address, 'label')
                     BMConfigParser().set(self.address, 'label', self.label)
                     BMConfigParser().save()
-                except:
-                    sqlExecute('''UPDATE addressbook set label=? WHERE address=?''', self.label, self.address)
+                except BaseException:
+                    sqlExecute(
+                        '''UPDATE addressbook set label=? WHERE address=?''',
+                        self.label,
+                        self.address)
             elif self.type == AccountMixin.SUBSCRIPTION:
                 from debug import logger
-                sqlExecute('''UPDATE subscriptions set label=? WHERE address=?''', self.label, self.address)
+                sqlExecute(
+                    '''UPDATE subscriptions set label=? WHERE address=?''',
+                    self.label,
+                    self.address)
             else:
                 pass
-        return super(Ui_AddressBookWidgetItem, self).setData(role, value)    
+        return super(Ui_AddressBookWidgetItem, self).setData(role, value)
 
-    def __lt__ (self, other):
+    def __lt__(self, other):
         if (isinstance(other, Ui_AddressBookWidgetItem)):
             reverse = False
-            if self.tableWidget().horizontalHeader().sortIndicatorOrder() == QtCore.Qt.DescendingOrder:
+            if self.tableWidget().horizontalHeader(
+            ).sortIndicatorOrder() == QtCore.Qt.DescendingOrder:
```

Comment:
This looks strange. Why not 
```
horizontalHeader()
.sortIndicatorOrder()
```
?

Reply:
I have this file also:

https://github.com/g1itch/PyBitmessage/commit/57a679812d130bfbf6ef66cd2f2d159e1c6d0b9e

```python
            reverse = QtCore.Qt.DescendingOrder == \
                self.tableWidget().horizontalHeader().sortIndicatorOrder()
```